toolbox/deps_update/main.go

```diff
@@ -175,7 +175,7 @@ func updateDependenciesOf(repo string) error {
 	if repo == istioRepo {
 		goPath := path.Join(repoDir, "../../..")
 		env := "GOPATH=" + goPath
-		if _, err = u.Shell(env + " make depend"); err != nil {
+		if _, err = u.Shell(env + " make depend.update"); err != nil {
```

Comment:
I can see how "depend.update" is apparently needed for force updates, but I'm not sure if "depend" is still needed for first-time init (I'm using the word "init" generically rather than making an reference to how "depend" is just an alias for "init").  Offhand also looks like init.sh is the only thing that pulls in a newer envoy tar.  So, you might have to do "make depend depend.update" until this situation with the makefiles is improved.

Reply:
depend is not needed, i just tested it the code we need to commit is the same, in both cases